examples/deployment-plus-min/README.md

```diff
@@ -0,0 +1,56 @@
+# Example
+
+In this example we deploy the NGINX Ingress Controller (edge) as a [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) using the NGINX Ingress Operator for NGINX Plus.
+
+## Prerequisites
+
+1. Have the NGINX Ingress Operator deployed in your cluster. Follow [installation](../../README.md#installation) steps.
+2. Build the NGINX Ingress Controller for Plus image and push it to a private repository following [these instructions](https://docs.nginx.com/nginx-ingress-controller/installation/building-ingress-controller-image/#building-the-image-and-pushing-it-to-the-private-registry). 
+
+**Note**: For the build process, if using Openshift, use the `DOCKERFILE=openshift/DockerfileForPlus` variable. 
```

Comment:
Do you need to have a similar comment in the OSS example?

Reply:
Yeah, I was waiting for the image to be pushed in the IC repository to update this example